### DISCUSSION TITLE
2016/07/03/beacon-tower-near-polkinghorne-spring-nevada

### DISCUSSION THREAD

**Discussion by giscus** [jengalas/thesurveystation-jekyll#10] [Announcements]
# 2016/07/03/beacon-tower-near-polkinghorne-spring-nevada

http://localhost:4000/2016/07/03/beacon-tower-near-polkinghorne-spring-nevada

**Comment by jengalas** [comment]
Posted by Harvey Hartman
January 23, 2017 at 12:24 pm
_Imported from WordPress_

Great Find! I’m surprised that the tower and it’s beacon are still intact after all these years. Any chance of getting some close-up pictures of the beacon added?

Harvey Hartman
Waller, TX
